feat: implement `access_token` auth method (#668)
ovh/terraform-provider-ovh: website/docs/index.html.markdown

You can find more details about the configuration parsing on repository [go-ovh](https://github.com/ovh/go-ovh).

### Access token

The provider will look for the token either at ``OVH_ACCESS_TOKEN`` environment
variable, or get it via ``access_token`` argument in the provider's stanza.

> Similarly to OAuth2 method, the endpoint might need to be configured (either via

Reviewer: ```suggestion
Similarly to OAuth2 method, the endpoint must be configured (either via
```
